spec/services/hyrax/resource_characterizer_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe Hyrax::ResourceCharacterizer do
+  subject(:characterizer) { described_class.new(source: resource) }
+
+  before do
+    allow(Hyrax::FileSet).to receive(:find).with(resource.id).and_return(resource)
+    allow(Hydra::Works::CharacterizationService).to receive(:run)
+    allow(Hyrax.persister).to receive(:save)
```

Comment:
okay to just actually call save on the persister?

if we want to avoid round tripping wings, we could do `allow(Hyrax).to receive(:metadata_adapter).and_return(Valkyrie::MetadataAdapter.find(:test_adapter))` to use the test/metadata adapter for the duration.

i've considered rigging this into some RSpec metadata, but it hasn't made it's way into the codebase yet.